frontend/src/lib/k8s/event.ts

```diff
@@ -164,6 +164,46 @@ class Event extends makeKubeObject<KubeEvent>('Event') {
     return objInstance;
   }
 
+  static useListQueryForClusters(
```

Comment:
It means that this hook will fail if the number of clusters change between renders.

To avoid that, the parent component should set the `key` for the component that uses this hook.
Using the `key` will remount the component when clusters change and hook wont fail.

You can see this in the `src/component/App/Home/index.tsx:170`

```tsx
  return <HomeComponent clusters={clusters} key={Object.keys(clusters).join('')} />;
```

HomeComponent uses `useWarningListQuery` hook and when clusters change, the key will also change, meaning the previous state of the HomeComponent will be discarded and it will be created from scratch.